_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -565,7 +565,9 @@
 	id = "AI"
```

Comment:
alright, starting a review for stuff I've found

Reply:
![image](https://github.com/user-attachments/assets/7ebc3a3f-7963-4bf8-96b8-64be4cd31205)
there's no way to open the armory shutters, the button is on a south wall and has no sprite as a result

Reply:
![image](https://github.com/user-attachments/assets/14430ddf-c01e-4d95-b1ee-4cd5263f743d)
same's true of the blast shutter buttons at the bridge captain's desk

Reply:
viro is inaccessible, first door's access button is both on a south wall and on the same tile as the airlock, so it can't be pressed, and the second door's is on the same tile as the airlock and hidden behind a poster